tests/framework/PostProcessors/Validation/tests

```diff
@@ -27,4 +27,25 @@
     rel_err = 0.00001
     zero_threshold = 1e-9
   [../]
+  [./test_validation_gate_pcm_StaticSeth]
+    type = 'RavenFramework'
+    input = 'test_validation_gate_pcm_StaticSeth.xml'
+    csv = 'PhysicsGuidedCoverageMapping/pcmStatic_metric_dump.csv'
+    rel_err = 0.00001
+    zero_threshold = 1e-9
+  [../]
+  [./test_validation_gate_pcm_SnapshotSeth]
+    type = 'RavenFramework'
+    input = 'test_validation_gate_pcm_SnapshotSeth.xml'
```

Comment:
The test name here is not right, please update it, please pay attention to the capital and lowercase letters